types/mui-datatables/index.d.ts

```diff
@@ -58,24 +57,24 @@ interface MUIDataTableTextLabelsBody {
 }
 
 interface MUIDataTableTextLabelsPagination {
+    displayRows: string;
     next: string;
     previous: string;
     rowsPerPage: string;
-    displayRows: string;
```

Comment:
Why is this re-ordered?

Reply:
Alphabetized properties inside interface